src/NServiceBus.AcceptanceTests/Performance/ProcessingOptimizations/When_limiting_concurrency_via_both_api_and_config.cs

```diff
@@ -7,18 +7,19 @@ namespace NServiceBus.AcceptanceTests.Config
     using NServiceBus.AcceptanceTests.FakeTransport;
     using NServiceBus.Config;
     using NUnit.Framework;
+    using ScenarioDescriptors;
 
     public class When_limiting_concurrency_via_both_api_and_config : NServiceBusAcceptanceTest
     {
         [Test]
         public async Task Should_throw()
         {
-            var context = await Scenario.Define<Context>()
+            await Scenario.Define<Context>()
                 .WithEndpoint<ThrottledEndpoint>(b => b.CustomConfig(c => c.LimitMessageProcessingConcurrencyTo(10)))
                 .Done(c => c.EndpointsStarted)
+                .Repeat(r => r.For(Transports.AllAvailable.SingleOrDefault(t => t.Key == "FakeTransport")))
```

Comment:
The root cause seems to be the way the Default scenario descriptor is working. If a certain env variable is set, then any other  UseTransport specified value will be ignored, and ultimately blow up. But even in the msmq oriented test runs it seems to rely on sheer luck, if there is more than 1 fake transport it would also fail https://github.com/Particular/NServiceBus/blame/develop/src/NServiceBus.AcceptanceTests/ScenarioDescriptors/Transports.cs#L39